backdoor/src/main/java/org/codice/acdebugger/backdoor/Backdoor.java

```diff
@@ -197,14 +205,14 @@ public String getDomainInfo(Object obj, Object permission) {
             (PrivilegedAction<String>)
                 () ->
                     JsonUtils.toJson(
-                        getDomainInfo((ProtectionDomain) obj, (Permission) permission)));
+                        getDomainInfo0((ProtectionDomain) obj, (Permission) permission)));
```

Comment:
ℹ️ The rename to add `0` is to avoid Spock calling the wrong method when testing